content/docs/setup/customize/aws-elb/alb.md

```diff
@@ -0,0 +1,185 @@
+{{< /callout >}}
+
+## Before you begin
+
+1. Create or use an existing AWS account. 
+2. Follow the [Get started guide](/docs/quickstart/) to install {{< reuse "docs/snippets/product-name.md" >}} and deploy the httpbin sample app. You do not need to set up a Gateway as you create a custom Gateway as part of this guide. 
+   
+## Step 1: Deploy gateway proxy resources
+ 
+1. Create a Gateway resource with an HTTP listener.
+   ```yaml
+   kubectl apply -n {{< reuse "docs/snippets/ns-system.md" >}} -f- <<EOF
+   kind: Gateway
+   apiVersion: gateway.networking.k8s.io/v1
+   metadata:
+     name: alb
+   spec:
+     gatewayClassName: kgateway
+     listeners:
+     - protocol: HTTP
+       port: 8080
+       name: http
+       allowedRoutes:
+         namespaces:
+           from: All
+   EOF
+   ```
+
+2. Create DirectResponse and HTTPRoute resources to configure a healthcheck for the gateway. This step is required for AWS ELB health checks to pass. 
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: gateway.kgateway.dev/v1alpha1
+   kind: DirectResponse
+   metadata:
+     name: httpbin-healthcheck-dr
+     namespace: httpbin
+   spec:
+     status: 200
+   ---
+   apiVersion: gateway.networking.k8s.io/v1
+   kind: HTTPRoute
+   metadata:
+     name: httpbin-healthcheck
+     namespace: httpbin
+     labels:
+       example: httpbin-route
+   spec:
+     parentRefs:
+       - name: alb
+         namespace: {{< reuse "docs/snippets/ns-system.md" >}}
+     rules:
+     - matches:
+       - path:
+           type: Exact
+           value: /healthz
+       filters:
+       - type: ExtensionRef
+         extensionRef:
+          name: httpbin-healthcheck-dr
+          group: gateway.kgateway.dev
+          kind: DirectResponse
+   EOF
+   ```
+
+3. Create an HTTPRoute resource to configure to open up a port for the `albtest.com` hostname on the gateway proxy.
+   ```yaml
+   kubectl apply -f- <<EOF
+   apiVersion: gateway.networking.k8s.io/v1
+   kind: HTTPRoute
+   metadata:
+     name: httpbin-alb
+     namespace: httpbin
+     labels:
+       example: httpbin-route
+   spec:
+     parentRefs:
+       - name: alb
+         namespace: {{< reuse "docs/snippets/ns-system.md" >}}
+     hostnames:
+       - "albtest.com"
+     rules:
+     - matches:
+       - path:
+           type: PathPrefix
+           value: /
+       backendRefs:
+       - name: httpbin
+         port: 8000
+   EOF
+   ```
+
+## Step 2: Create an ALB with the AWS Load Balancer controller
+
+{{< reuse "docs/snippets/aws-elb-controller-install.md" >}}
+
+5. Use an Ingress resource to define your ALB. When you apply this resource, the AWS Load Balancer Controller creates the ALB in your account.
```

Comment:
numbering is off. not sure if it shows on the actual page though

Reply:
aaah ok that is explains why the gateway does not create an NLB up until this step :D